src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -82,6 +82,13 @@ GraphingCalculator::GraphingCalculator()
     virtualKey->Key = (VirtualKey)187; // OemAdd key
     virtualKey->Modifiers = VirtualKeyModifiers::Control;
     ZoomInButton->KeyboardAccelerators->Append(virtualKey);
+
+    // add shadow to the trace pointer when not in high contrast mode
+    auto accessibilitySettings = ref new Windows::UI::ViewManagement::AccessibilitySettings();
+    if (!accessibilitySettings->HighContrast)
```

Comment:
I don't think this takes into account switching to high contrast while the app is running

Reply:
good catch.  I'll fix that.